perf_test/blas/blas3/KokkosBlas3_trmm_perf_test.hpp

```diff
@@ -292,7 +292,7 @@ void __do_trmm_serial_batched(options_t options, trmm_args_t trmm_args) {
   return;
 }
 
-#if !defined(KOKKOS_ENABLE_CUDA)
+#if !defined(KOKKOS_ENABLE_CUDA) && !defined(KOKKOS_ENABLE_HIP)
```

Comment:
Are these basically "host only" checks ? Then we can add one check as host only and not worry about this for Intel SYCL and OpenMP target backends.

Reply:
Good point, we can definitely create a KOKKOSKERNELS_HOST_ONLY macro that stores that information and is used in the code.

Reply:
see issue #843